registrations/admin.py

```diff
@@ -87,3 +100,32 @@ def get_readonly_fields(self, request, obj=None):
 
 
 admin.site.register(Registration, RegistrationAdmin)
+
+
+class PriceGroupAdmin(VersionAdmin):
+    fields = (
+        "id",
+        "publisher",
+        "description",
+    )
+    list_display = (
+        "id",
+        "publisher",
+        "description",
+    )
+    list_filter = (PublisherFilter,)
+    search_fields = ("description",)
+
+    def save_model(self, request, obj, form, change):
+        if obj.pk is None:
+            obj.created_by = request.user
+        else:
+            obj.last_modified_by = request.user
```

Comment:
/events/admin.py file has BaseAdmin class. Could this be inherited from there?

Reply:
`BaseAdmin` uses a `modified_by` field, but in registration models we use `last_modified_by` instead. Because of this it might not be worth it to inherit.

Reply:
Ah, very true. The naming difference is a little bit confusing. But there is no reason to do anything to it right now.

Reply:
There is a similar save_model method in RegistrationAdmin as well in you like to remove some duplicate code.

Reply:
I created a new base class in `registrations/admin.py`